app/controllers/DevelopersController.scala

```diff
@@ -85,14 +83,14 @@ trait DevelopersController extends FrontendController with GatekeeperAuthWrapper
       val pageNumber = optionalPageNumber.getOrElse(1)
       
       for {
-        apps <- developerService.fetchApplications(apiFilter)
+        apps <- applicationService.fetchApplications(apiFilter)
         apis <- apiDefinitionConnector.fetchAll
         filterOps = (developerService.filterUsersBy(apiFilter, apps) _
-                     andThen developerService.filterUsersBy(statusFilter) _)
-        devs <- developerService.fetchDevelopers
-        users = filterOps(devs)
-        emails = developerService.emailList(users)
-        page = PageableCollection(users, pageNumber, pageSize)
+                     andThen developerService.filterUsersBy(statusFilter))
+        allUsers <- developerService.fetchDevelopers
+        filteredUsers = filterOps(allUsers)
+        emails = Developers(filteredUsers).emailList
```

Comment:
the email to button no longer exists, so this line should probably be removed along with any references